tests/Functional/RedisCommandsTest.php

```diff
@@ -18,6 +18,8 @@ class RedisCommandsTest extends TerminusTestBase
      */
     public function testRedisEnable()
     {
+        // This usually runs right after site plan change, let's sleep for a bit before continuing.
+        sleep(5);
```

Comment:
side-comment -- seems like the appropriate place for this `sleep` would be between the tests, but OOP/inversion-of-control prevents us from putting it there. <grumble grumble>

Seems like it might make more sense for this to be at the end of the site plan change rather than the beginning of the test that runs after it.  Are we able to put it there, or does the code structure prevent that?